### PR TITLE
Create app.tar.gz using a temporary dir when necessary

### PR DESCRIPTION
**Pull request by aguillenv**

### CODE REVIEW

File: anycloud/cli/src/common/mod.rs [hunk 1]
```diff
@@ -2,22 +2,40 @@ use std::env;
...
+        .output()
+        .unwrap();
+      println!("{:?}", output);
+      match read(format!("{}/{}", file_path, file_name)) {
+        Ok(file) => Ok(file),
+        Err(_) => Err(format!("No {} at {}", file_name, file_path).into()),
+      }
+    }
+    None => {
+      let pwd = env::current_dir();
```
Comment (dfellis) [inline anycloud/cli/src/common/mod.rs:22]: Why not just expect the full path name for a file and have the consumer code use this API directly?

File: anycloud/cli/src/common/mod.rs [hunk 2]
```diff
@@ -2,22 +2,40 @@ use std::env;
...
+
+async fn get_file(file_name: &str, file_path: Option<&str>) -> Result<Vec<u8>, String> {
+  match file_path {
+    Some(file_path) => {
+      println!("{}", format!("{}/{}", file_path, file_name));
+      let output = Command::new("ls")
+        .arg(format!("{}/{}", file_path, file_name))
+        .output()
+        .unwrap();
+      println!("{:?}", output);
```
Comment (cdmistman) [inline anycloud/cli/src/common/mod.rs:15]: any reason for these prints or for the `ls`?

Comment (aguillenv) [reply anycloud/cli/src/common/mod.rs:15]: No reason at all, I used it for testing and forgot to remove it

File: anycloud/cli/src/common/mod.rs (hunk 1 above)
Comment (aguillenv) [reply anycloud/cli/src/common/mod.rs:22]: I was trying to follow the same pattern as it was to avoid the `match` for the `pwd` every time we call it.

**Review by cdmistman — APPROVED** [review]

**Review by dfellis — APPROVED** [review]

**Pull request merged**